exercises/atbash_cipher/example.cr

```diff
@@ -0,0 +1,13 @@
+module Atbash
+  MAPPING = (alp = ('a'..'z').to_a).zip(alp.reverse).to_h
+  extend self
+  def encode(message : String) : String
+    message = message.downcase.gsub(/[^a-z0-9]/, "")
+    encoded_chars = message.chars.map{|c| MAPPING[c]? || c}
+    encoded_string = ""
+    encoded_chars.each_with_index do |c, i|
+      encoded_string += ((i%5 == 0 ? " " : "") + c)
+    end
+    encoded_string.strip
+  end
```

Comment:
It might be worth splitting up the individual functions into their own methods and maybe putting this into a class. Up to you... but I think it would be cleaner.

Like:
```crystal
def encoded_chars(message)
  message.chars.map{|c| MAPPING[c]? || c}
end
```

Or something.

Just looking at the Ruby implementation...